disperser/batcher/metrics.go

```diff
@@ -288,7 +297,14 @@ func (t *DispatcherMetrics) ObserveLatency(operatorId string, success bool, late
 	if !success {
 		label = "failure"
 	}
-	t.Latency.WithLabelValues(operatorId, label).Observe(latencyMS)
+	// The Latency metric has "operator_id" but we null it out because it's separately
+	// tracked in OperatorLatency.
+	t.Latency.WithLabelValues("", label).Observe(latencyMS)
```

Comment:
do we need the label if we are just going to null it out?

Reply:
Haven't evaluated it much, is it safe to delete the label for this?

Reply:
I don't see anything obvious that would go wrong. Maybe test in preprod?

Reply:
It'd be better to keep it for a while, we may want to look back the operator latency in past month for example, if this label is gone, we cannot do that.